common_utils_py/agreements/service_agreement.py

```diff
@@ -81,6 +87,14 @@ def get_number_nfts(self):
         """
         return int(self.get_param_value_by_name('_numberNfts'))
 
+    def get_key_hash(self):
+        """
+        Return the key hash
+
+        :return: Str
+        """
+        return int(self.get_param_value_by_name('_numberNfts'))
```

Comment:
shouldn't be `_hash`?

Reply:
Yeah, there's some cleanup left to do...